app/src/main/java/me/magnum/melonds/MelonDSApplication.kt

```diff
@@ -35,13 +32,12 @@ class MelonDSApplication : Application(), Configuration.Provider {
     }
 
     private fun createNotificationChannels() {
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
-            return
-        }
+        val defaultChannel = NotificationChannelCompat.Builder(
+            NOTIFICATION_CHANNEL_ID_BACKGROUND_TASKS, NotificationManagerCompat.IMPORTANCE_LOW)
```

Comment:
Everything seems fine here. I'll just ask you to put the parameters in the same line as the constructor call.

```kotlin
val defaultChannel = NotificationChannelCompat.Builder(NOTIFICATION_CHANNEL_ID_BACKGROUND_TASKS, NotificationManagerCompat.IMPORTANCE_LOW)
            .setName(getString(R.string.notification_channel_background_tasks))
            ...
```

Reply:
Sure thing!